src/common/transformations/include/transformations/op_conversions/convert_convolution.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/matcher_pass.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API ConvertConvolution;
+
+}  // namespace pass
+}  // namespace ov
+
+class ov::pass::ConvertConvolution : public ov::pass::MatcherPass {
+public:
+    OPENVINO_MATCHER_PASS_RTTI("ConvertConvolution");
```

Comment:
`ConvertConvolution` is too abstract name for transformation. Maybe `ConvolutionWithBiasFusion` ?

Reply:
Agree, aligned with the similar pass and named "ConvolutionBiasFusion"